dev.py

```diff
@@ -54,7 +54,7 @@
 setuptools_path = os.path.dirname(os.path.dirname(setuptools.__file__))
 
 ######################################################################
-# Install buildout
+print("Installing buildout ...")
```

Comment:
Is this here on purpose or leftover from debugging?

Reply:
It is mostly a leftover from debugging. I thought it could be helpful in the future, too.